sdk/opendp/smartnoise/sql/private_reader.py

```diff
@@ -202,7 +203,7 @@ def _execute_ast(self, query, cache_exact=False):
             kc_pos = kcc_pos.pop()
 
         # make a list of mechanisms in column order
-        mechs = [Gaussian(self.epsilon_per_column, self.delta, s, max_contrib, self.interval_widths) if s is not None else None for s in sens]
```

Comment:
we should keep test_gauss?

Reply:
test_gauss was testing the proper noise distribution when sensitivity changed.  That's handled in core library tests now